src/main/java/org/broadinstitute/hellbender/utils/downsampling/ReservoirDownsampler.java

```diff
@@ -54,37 +39,21 @@
 
     /**
      * Construct a ReservoirDownsampler
+     *  @param targetSampleSize Size of the reservoir used by this downsampler.
```

Comment:
"Size of the reservoir used by this downsampler. Number of items retained after downsampling will be min(totalReads, targetSampleSize)"